circuitpython_library_pypi_stats.md

```diff
@@ -3,6 +3,10 @@
 Here is a listing of current Adafruit CircuitPython libraries download statistics.  
 There are 308 libraries available.
 
+
+Total Blinka downloads: 34218  
+Total PyPI library downloads: 182737
```

Comment:
These should be a heading 3 (`###`), and maybe make the numbers bold. (Try it and see how it looks, and make a decision on it.)